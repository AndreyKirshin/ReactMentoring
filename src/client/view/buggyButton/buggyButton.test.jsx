import React from 'react';
import { shallow } from 'enzyme';
import BuggyButton from './buggyButton';


describe('BuggyButton component', () => {

    it('should be render correctly', () => {
        const component = shallow(
            <BuggyButton />);
        expect(component).toMatchSnapshot();
    });
});