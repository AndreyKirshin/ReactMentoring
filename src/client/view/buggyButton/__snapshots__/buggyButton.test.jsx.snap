// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BuggyButton component should be render correctly 1`] = `ShallowWrapper {}`;
